tycho-surefire/tycho-surefire-plugin/src/main/java/org/eclipse/tycho/surefire/AbstractTestMojo.java

```diff
@@ -432,6 +432,9 @@ public abstract class AbstractTestMojo extends AbstractMojo {
     @Parameter(property = "surefire.rerunFailingTestsCount", defaultValue = "0")
     private Integer rerunFailingTestsCount;
 
+    @Parameter(defaultValue = "default")
+    private String tychoReportingMode;
```

Comment:
Maybe should be called `surefireReportingMode` but as it's Tycho specific I thought best to prefix with Tycho as I wasn't sure if it was also passed into Surefire too but looking at `OsgiSurefireBooter` now, it probably isn't.

Reply:
Is this necessary to have it an option here? Can't we just pick the best one according to current context?
More options means more user support and more maintenance care. So if there is no obvious reason to have one, it's better to avoid them.

Reply:
Yes it would be simpler.

 It's just in case we needed the flexibility of using e.g. the default reporting with JUnit5 (as is the current behaviour with tycho-surefire and maven-surefire). I'm not sure if there really is a requirement for that though, and the parameter in this form will not support the full flexibility of the maven-surefire plugin anyway.